roles/install-devstack/tasks/main.yml

```diff
@@ -27,6 +27,14 @@
     cmd: |
       set -e
       set -x
+      # As openstack/requirements already upgrade requests to 2.22.0,
+      # but didn't provider a 2.22.0 version for python34 which we used.
+      # Install python3.5
+      add-apt-repository ppa:deadsnakes/ppa -y
+      apt-get update
+      apt-get install python3.5 python3.5-dev libssl-dev -y
+      sudo update-alternatives --install $(which python3) python3 $(which python3.5) 1
+
```

Comment:
I find there are at least 3 places installing python3.5 3.6 or 3.7 in the repo. We can add a new common  role for it.

Reply:
Yeah, you, kevin and me already hit the issues.